### ISSUE TITLE
Post-Installation Tests Fail on HPC Cluster

### ISSUE THREAD

**Issue by mefuller** [ReactionMechanismGenerator/RMG-Py#1908]
I am attempting to install and run RMG locally on my university's HPC cluster (CentOS 7.7).
Compilation of RMG appears to work just fine, but most of the tests fail.

The errors from the failed tests are one of the following:

    ERROR: Failure: AttributeError (type object 'rmgpy.kinetics.model.TunnelingModel' has no attribute 'reduce_cython')
    ERROR: Failure: ImportError (/rwthfs/rz/cluster/home/db011307/RMG/RMG-Py/rmgpy/species.cpython-37m-x86_64-linux-gnu.so: undefined symbol: _intel_fast_memcmp)
    ERROR: Failure: ImportError (/rwthfs/rz/cluster/home/db011307/RMG/RMG-Py/rmgpy/chemkin.cpython-37m-x86_64-linux-gnu.so: undefined symbol: _intel_fast_memmove)

### Bug Description
Post-installation tests fail (see above)

### How To Reproduce
Installation per https://reactionmechanismgenerator.github.io/RMG-Py/users/rmg/installation/anacondaDeveloper.html

make test (fails, as above)

### Expected Behavior
Tests complete successfully

### Installation Information
Describe your installation method and system information.
 - OS: CentOS 7.7
 - Installation method: anaconda / compile RMG from source
 - RMG version information:
   - RMG-Py: 3.0.0-48-g2de5cf2
   - RMG-database: 3.0.0-16-gd626e2b

**Comment by mjohnson541** [comment]
Pretty sure this is a known issue with running RMG on CentOS. I think people have managed to run RMG on CentOS using Docker images before. @amarkpayne do you know more about this?

**Comment by amarkpayne** [comment]
Yes, in the past others have also tried to install on CentOS and have run into problems trying to compile RMG. While we aren't sure exactly what is causing this on CentOS, I believe others who have run into this problem have been able to successfully run RMG in a docker image. I believe the latest docker image is located at https://hub.docker.com/r/mliu49/rmg . @mliu49 let me know if this is not the case.

I would try running RMG using this docker image first, as it should be the easiest route going forward by far

**Comment by mliu49** [comment]
The latest version is at https://hub.docker.com/r/reactionmechanismgenerator/rmg. There's a build for the 3.0 release.

**Comment by JacksonBurns** [comment]
Docker is now the primary distribution method for RMG and resolving Cython compilation errors on HPC clusters is beyond our capabilities. Closing as resolved.
